wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/platform.kt

```diff
@@ -15,12 +15,27 @@
  */
 package com.squareup.wire.internal
 
+import com.squareup.wire.GrpcResponse
 import java.lang.reflect.Method
+import okio.IOException
 import okio.Sink
 import okio.Source
 import okio.gzip
 
-actual typealias Call = okhttp3.Call
```

Comment:
This typealias no longer works either, since `okhttp3.Call` references `okhttp3.Response`, which is no longer our implementation of `GrpcResponse`.